Sources/SKTestSupport/SkipUnless.swift

```diff
@@ -393,6 +393,7 @@ package actor SkipUnless {
     file: StaticString = #filePath,
     line: UInt = #line
   ) async throws {
+    #if os(Windows)  // docc is supported on Linux and Windows
```

Comment:
Why is this needed? And presumably the comment is incorrect 🤔?

Reply:
Clarified the comment

> docc is supported on macOS and Linux. Do not perform a check on those platforms to avoid accidentally skipping the tests in CI on those platforms.